requirements/_tests.txt

```diff
@@ -20,3 +20,5 @@ cloudpickle >1.3, <=3.1.0
 scikit-learn ==1.2.*; python_version < "3.9"
 scikit-learn ==1.5.*; python_version > "3.8"  # we do not use `> =` because of oldest replcement
 cachier ==3.1.2
+
+matplotlib==3.9.3
```

Comment:
why the pin here? we have this range:
https://github.com/Lightning-AI/torchmetrics/blob/f1d8f531fc5cb1120fe463930f7cf909ee9c4617/requirements/visual.txt#L4